src/containers/blocks.jsx

```diff
@@ -91,6 +113,7 @@ class Blocks extends React.Component {
         this.props.vm.addListener('VISUAL_REPORT', this.onVisualReport);
         this.props.vm.addListener('workspaceUpdate', this.onWorkspaceUpdate);
         this.props.vm.addListener('targetsUpdate', this.onTargetsUpdate);
+        this.props.vm.addListener('EXTENSION_WAS_ADDED', this.onExtensionWasAdded);
```

Comment:
This might just be me, but the word "was" in this event and related handlers seems unneeded. Could it just be `EXTENSION_ADDED` and `onExtensionAdded`?